.ci/docker-compose-file/docker-compose-datalayers-tcp.yaml

```diff
@@ -0,0 +1,13 @@
+version: '3.9'
+
+services:
+  datalayers_server_tcp:
+    container_name: datalayers_tcp
+    image: datalayers/datalayers:${DATALAYERS_TAG}
+    expose:
+      - "8361"
+    environment:
+      DATALAYERS_SERVER__TIMEZONE: "Etc/GMT"
```

Comment:
Q: why this zone ?

Reply:
It is actually to guarantee they have the same time string (i.e: 2024-08-20T22:09:03.136418Z) between the API query result with `calendar:system_time_to_rfc3339` in tests

https://github.com/HJianBo/emqx/blob/1a572cb93033c1b2047de99f0fc573251bfa359e/apps/emqx_bridge_datalayers/test/emqx_bridge_datalayers_SUITE.erl#L566